src/data/layer-2/layer-2.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Meta note, I think we as a team should probably make a decision on how to approach data files & what format is best, e.g. JSON vs. CSV. I think both have pros & cons.

Reply:
Agreed, ill book something to have a quick huddle next week. Im fine with either, just prefer JSON so went with that for draft.